zingo-sync/src/primitives.rs

```diff
@@ -35,6 +35,15 @@ impl SyncState {
             spend_locations: Vec::new(),
         }
     }
+
```

Comment:
From local checks this fn could currently be `pub(crate)`..  should we promote it to fully `pub` "on demand"?

Do we already have a consumer using it?

Reply:
I thought this could be useful API but Im happy with making it pub(crate), i will make this change on the following PR. I think it's a good policy to keep things private unless there is a known use case for sure